source/resampling_with_code2.Rmd

```diff
@@ -0,0 +1,669 @@
+a Hypothetical County jury will be Black.  But, if we select each juror at
+random from the population, that means that sometimes, by chance, we will have
+fewer than 3 Black jurors, and sometimes will have more than 3 Black jurors.
+And, by chance, sometimes we will have no Black jurors.  But, if the jurors
+really are selected at random, how often would we expect this to happen — that
+there are no Black jurors.  We would like to estimate the *probability* that
+we will get no Black jurors.  If that probability is small, then we have some
+evidence that the disparity in selection between Black and White jurors, was
+not "small".
+
+## One trial
+
+Remember the six steps:
+
+1. *We decide what we mean by one trial*.
+2. Work out the *outcome* of interest from the trial.
+2. Work out how to *simulate one trial*.
+3. *Repeat* the simulated trial procedure.
+4. *Count* the number of trials with an outcome that matches the outcome we
+   are interested in.
+5. Calculate the *proportion*.
+
+Are task in step 1 is made a little easier by the fact that our *trial* (in the resampling sense) is a simulated *trial* (in the legal sense).
+
+One trial, is 12 simulated jurors, each labeled by race (White or Black). For
+step 2, the outcome we are interested in is the number of Black jurors.
+
+Now comes the harder part.  How do we simulate one trial?
+
+One trial is 12 jurors, and we are interested only the race of each juror.  In
+our hypothesized world, where selection by race is entirely random, each juror
+has a 26% chance of being Black.
+
+We need a way of simulating a 26% chance.
+
+One way of doing this is by getting a random number from 0 through 99
+(inclusive).  There are 100 number in the range 0 through 99 (inclusive).
+
+::: {.callout-note}
+## Pick a number from 1 through 5
+
+Ranges can be confusing in normal speech because it is not always clear
+whether they include their beginning and end.  For example, if someone says
+"pick a number between 1 and 5", do they mean *all* the numbers, including the
+first and last (any of 1 or 2 or 3 or 4 or 5)?  Or do they mean only the
+numbers that are *between* 1 and 5 (so 2 or 3 or 4).  Or do they mean all the
+numbers up to, but not including 5 (so 1 or 2 or 3 or 4).
+
+To avoid this confusion, we will nearly always use "from" and "through" in
+ranges, meaning that we do include both the start and the end number.  For
+example, if we say "pick a number from 1 through 5" we mean one of 1 or 2 or 3
+or 4 or 5.
+"""
+
+We will arbitrarily say that the juror is White if the random number is in the
+range from 0 through 73.  74 of the 100 numbers are in this range, so
+the juror has a 74/100 = 74% chance of getting the label "White".  We will say
+the juror is Black if the random number is in the range 74 though 99.  There
+are 26 such numbers, so the juror has a 26% chance of getting the label
+"Black".
+
+Next we need a way of getting a random number in the range 0 through 99. One
```

Comment:
This is a bit confusing, since we don't do the same on a computer.  Perhaps you can say: if you had to do it by hand, you could do it like this, but fortunately it is easy to simulate a 100-sided die on a computer.

Reply:
Have put something like this in.